TLM/TLM/Patch/_PathManager/CreatePathPatch.cs

```diff
@@ -38,23 +39,28 @@ public class CreatePathPatch {
             NetInfo.LaneType laneTypes, VehicleInfo.VehicleType vehicleTypes, float maxLength, bool isHeavyVehicle,
             bool ignoreBlocked, bool stablePath, bool skipQueue, bool randomParking, bool ignoreFlooded, bool combustionEngine, bool ignoreCost)
         {
-            if (VehicleID == 0) {
-                Log.Error("unexpected VehcileID == 0. " +
-                    "StartPathFind() is not ptched or it failed to set VehicleID. " +
-                    "Falling back to Vanilla path find");
-                return true; // fall back to Vanilla path find.
-            }
+            PathCreationArgs args = default;
```

Comment:
mmmh where?

Reply:
My fault, I thought that due to initializing with default first, at then replacing the values later instead if directly initializing with the expected values it would be more work.
But I checked in sharplab and I am wrong, so forget about it.